acdc-ws/app/utils/MetricReporter.scala

```diff
@@ -0,0 +1,105 @@
+package utils
+
+import io.prometheus.client._
+import io.prometheus.client.exporter.common.TextFormat
+import io.prometheus.client.hotspot.DefaultExports
+
+import java.io.StringWriter
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.jdk.CollectionConverters._
+
+object Metrics {
+
+  val DefaultBucketValues: Array[Double] = Array(
+    .005,
+    .010,
+    .025,
+    .050,
+    .100,
+    .250,
+    .500,
+    1.0,
+    2.5,
+    5.0,
+    10.0,
+    15.0,
+    60.0,
+    300.0,
+    1200.0,
+    Double.PositiveInfinity
+  )
+
+  val httpDurationSeconds: Histogram = Histogram.build
+    .buckets(DefaultBucketValues: _*)
+    .name("http_request_duration_seconds")
+    .help("Total HTTP Requests Count")
+    .register
+
+  val httpTotalRequests: Counter = Counter.build
+    .name("http_requests_total")
+    .help("Total HTTP Requests Count")
+    .labelNames("status")
+    .register
+
+  val apiLatencySummary: Summary = Summary
+    .build()
+    .name("apiLatencySummary")
+    .labelNames("path", "arguments", "method")
+    .help("Profile API response time summary")
```

Comment:
define some percentiles for this Summary

Reply:
Is this related to what you showed me yesterday?  Let's touch base on slack on what this standardization looks like.

Reply:
An example 
```
  val httpDurationPercentiles: Summary = Summary
    .build()
    .name("http_request_percentile_seconds")
    .help("Total HTTP Requests Duration Percentile in seconds")
    .quantile(0.5d, 0.001d)
    .quantile(0.95d, 0.001d)
    .quantile(0.99d, 0.001d)
    .register()
```

Reply:
Agree and done.

Reply:
Agree and done.